src/features/orders/Orders.tsx

```diff
@@ -60,20 +66,24 @@ export class Orders extends React.Component<Props, State> {
       </>
     );
   }
+  private get swipebleOptions() {
```

Comment:
@DiamondRock597 You copied this from code sample (where we handle this with RTL/LTR), you can put it directly to props.